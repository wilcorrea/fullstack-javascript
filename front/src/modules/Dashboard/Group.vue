<template>
  <!-- I'm no kidding, it's just a hack -->
  <!-- It's used to put routes sisters in different levels -->
  <!-- @see front/src/domains/Order/routeFile.ts:28 -->
  <router-view />
</template>
