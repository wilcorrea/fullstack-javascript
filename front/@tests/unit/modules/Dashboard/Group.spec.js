import { shallowMount } from '@vue/test-utils'
import Group from 'src/modules/Dashboard/Group.vue'

describe('Group', () => {
  const RouterView = { template: '<div>RouterView</div>' }

  const wrapper = shallowMount(Group, { components: { RouterView } })

  /**
   * @test
   */
  it('check component markup', () => {
    expect(wrapper.html()).toMatch('')
  })

  /**
   * @test
   */
  test('should render content correctly', () => {
    expect(wrapper.html()).toMatchSnapshot()
  })
})
