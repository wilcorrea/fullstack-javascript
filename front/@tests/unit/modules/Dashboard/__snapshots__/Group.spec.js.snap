// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Group should render content correctly 1`] = `<router-view-stub></router-view-stub>`;
